src/js_sync.hpp

```diff
@@ -170,6 +171,10 @@ class SessionClass : public ClassDefinition<T, WeakSession> {
 
     static void simulate_error(ContextType, FunctionType, ObjectType, size_t, const ValueType[], ReturnValue &);
     static void refresh_access_token(ContextType, FunctionType, ObjectType, size_t, const ValueType[], ReturnValue &);
+#if REALM_ENABLE_SYNC
```

Comment:
`src/js_sync.hpp` is only included if `REALM_ENABLE_SYNC` is defined so I think we don't need this.

Reply:
will remove